docs/contribute/locally.md

```diff
@@ -35,7 +35,7 @@ This guide uses the first option. If you'd like to clone the repository and buil
    Run this command to download and install the latest version of `docs-builder`:
 
    ```sh
-   sudo curl -L https://raw.githubusercontent.com/elastic/docs-builder/refs/heads/main/install.sh | sh
+   curl -sL https://ela.st/docs-builder-install | sudo sh
```

Comment:
Just to flag [my comment](https://github.com/elastic/docs-builder/pull/1244#discussion_r2084155287) (that was collapsed due to referring to the old line) still stands

Reply:
Would using a different, user-level path, thus removing the need for sudo help here?

Reply:
That and not using URL shorteners?

Reply:
I've removed sudo from the command, as the user will be prompted anyway when required. Let's continue the conversation in the other comment.